build.gradle

```diff
@@ -41,6 +41,10 @@ task validateLocalDocSite(type: Exec) {
 quality.dependsOn validateLocalDocSite
 
 subprojects {
+  // mockito 5 only supports jdk11+
+  if (JavaVersion.current() < JavaVersion.VERSION_11) {
+    project.setProperty("mockitoCoreVersion", "4.11.0")
```

Comment:
If 4.x works well on JDK21, will be preferable to stick to that until we can bump the minimal required version for ST to JDK11. Using 4.x conditionally opens a risk of compilation errors if 5.x breaks API.
If there are any issues with 4.x on JDK21, consider adding an entry in gradle.properties for 4.x. If it's only here, it's easy to forget.

Reply:
4.x doesn't work due to bytebuddy and incompatible class file structure

Reply:
I can create another property for this